tensorpack/dataflow/serialize.py

```diff
@@ -152,6 +154,46 @@ def load(path, size=None):
         return ds
 
 
+class HDF5Serializer():
+    """
+    Write datapoints to a HDF5 file.
+
+    Note that HDF5 files are in fact not very performant and currently do not support lazy loading.
+    It's better to use :class:`LMDBSerializer`.
+    """
+    @staticmethod
+    def save(df, path, data_paths=[]):
+        """
+        Args:
+            df (DataFlow): the DataFlow to serialize.
+            path (str): output hdf5 file.
+            data_paths (list, optional): list of h5 paths to zipped.
```

Comment:
why is it optional and has a default value? from the code it looks like it is necessary and has to have the same length as datapoints.

Reply:
Good catch!